.github/workflows/ci.yml

```diff
@@ -19,62 +19,128 @@ jobs:
         include:
           - operating-system: 'ubuntu-20.04'
             php-version: '7.4'
-            job-description: 'with lowest deps'
+            job-description: 'Fixer with lowest deps'
+            run-fixer: 'yes'
+            run-phpdoc-to-native-type: 'yes' # should be checked on the lowest supported PHP version
             composer-flags: '--prefer-lowest' # should be checked on the lowest supported PHP version
+            execute-flex-with-symfony-version: '^5' # explicit check for Symfony 5.x compatibility
 
           - operating-system: 'ubuntu-20.04'
             php-version: '7.4'
-            job-description: 'with PHPDoc to type rules'
-            phpdoc-to-type-rules: 'yes' # should be checked on the lowest supported PHP version
+            job-description: 'tests with lowest deps'
+            run-tests: 'yes'
+            composer-flags: '--prefer-lowest' # should be checked on the lowest supported PHP version
+            execute-flex-with-symfony-version: '^5' # explicit check for Symfony 5.x compatibility
 
           - operating-system: 'ubuntu-20.04'
             php-version: '7.4'
-            job-description: 'with Symfony ^5'
-            execute-flex-with-symfony-version: '^5' # explicit check for Symfony 5.x compatibility
+            job-description: 'Fixer'
+            run-fixer: 'yes'
+            run-phpdoc-to-native-type: 'yes' # should be checked on the lowest supported PHP version
+
+          - operating-system: 'ubuntu-20.04'
+            php-version: '7.4'
+            job-description: 'tests'
+            run-tests: 'yes'
```

Comment:
imho it's enough to run 7.4 --with lowest` and no need to run 7.4 (with highest)